pkg/client/cache/reflector.go

```diff
@@ -161,7 +161,8 @@ func (r *Reflector) listAndWatch(stopCh <-chan struct{}) {
 			default:
 				glog.Errorf("Failed to watch %v: %v", r.expectedType, err)
 			}
-			return
+			time.Sleep(time.Second)
```

Comment:
Errors from etcd are reported in line 206, not here. Those reported here are only on the connection level.